packages/@expo/fingerprint/src/sourcer/Expo.ts

```diff
@@ -26,7 +26,8 @@ export async function getExpoConfigSourcesAsync(
   let config: ProjectConfig;
   let expoConfig: ExpoConfig;
   let loadedModules: string[] = [];
-  const ignoredFile = await createTempIgnoredFileAsync(options);
+  const tmpDir = await fs.mkdtemp(path.join(os.tmpdir(), 'expo-fingerprint-'));
```

Comment:
are these changes to ignored files separate/unrelated?

Reply:
it's just fix the fact that i didn't use the mkdtemp dir
https://github.com/expo/expo/blob/e456134de4e15fecfa27fad76c7280832c994db4/packages/%40expo/fingerprint/src/sourcer/Expo.ts#L182-L183